sim/rogue/assassination/assassination_test.go

```diff
@@ -22,7 +22,7 @@ func TestAssassination(t *testing.T) {
 		OtherGearSets: []core.GearSetCombo{
 			core.GetGearSet("../../../ui/rogue/assassination/gear_sets", "p3_assassination"),
 			core.GetGearSet("../../../ui/rogue/assassination/gear_sets", "preraid_assassination"),
```

Comment:
We don't really need to run tests for all sets unless they are explicitly testing individual things ofc. Can Preraid be removed?